testsuite/features/secondary/minxen_guests.feature

```diff
@@ -218,13 +212,21 @@ Feature: Be able to manage XEN virtual machines via the GUI
     And I enter "512" as "memory"
     And I enter "/var/testsuite-data/disk-image-template.qcow2" as "disk0_source_template"
     And I select "test-net0" from "network0_source"
+    And I select "Spice" from "graphicsType"
     And I click on "Create"
     Then I should see a "Hosted Virtual Systems" text
     When I wait until I see "test-vm3" text
     And I wait at most 500 seconds until table row for "test-vm3" contains button "Stop"
     And "test-vm3" virtual machine on "xen_server" should have 512MB memory and 1 vcpus
     And "test-vm3" virtual machine on "xen_server" should have 1 NIC using "test-net0" network
-    And "test-vm3" virtual machine on "xen_server" should have a "test-vm3_system" xen disk from pool "test-pool0"
+    And "test-vm3" virtual machine on "xen_server" should have a "/var/lib/libvirt/images/test-pool0/test-vm3_system" xen disk
+
+@virthost_xen
+  Scenario: Show the Spice graphical console for Xen
+    Given I am on the "Virtualization" page of this "xen_server"
+    When I click on "Graphical Console" in row "test-vm3"
+    And I switch to last opened window
+    Then I wait until I see the spice graphical console
```

Comment:
Could be "And"

Reply:
Actually, I prefer it as Then :D I think we agreed to don't consider the "I wait until" part, sometime in the past.